app/views/admin/companies/edit.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= form_for [:admin, @company] do |f| %>
+<div class="form-group">
+  <%= f.label(:name, "Name (First and Last)") %>
+  <%= f.text_field :name, required: true %><br>
+</div>
+<div class="form-group">
+  <%= f.label(:organization, "Organization") %>
+  <%= f.text_field :organization, required: true  %><br>
+</div>
+<div class="form-group">
+  <%= f.label(:title, "Title") %>
+  <%= f.text_field :title, required: true  %><br>
+</div>
+<div class="form-group">
+  <%= f.label(:state, "State") %>
+  <%= f.select :state, options_for_select(state_helper), placeholder: "AL" %>
+</div>
+<div class="form-group">
+  <%= f.label(:city, "City") %>
+  <%= f.select :city, options_for_select(city_helper), placeholder: "Please select..." %><br>
```

Comment:
this is gonna break, I took out the city_helper method and am querying database now.